tests/templates.test.ts

```diff
@@ -64,7 +64,7 @@ const testCase = async (template: string, done: () => void) => {
 
     browser.then(async (browser) => {
       const page = await browser.newPage();
-      await page.goto("http://localhost:3000");
+      await page.goto("http://localhost:3000/admin");
```

Comment:
Almost, what I suggest is to add `base` parameter to `testCase` function, that's set to `/` by default.

```js
const testCase = async (template: string, done: () => void, base = "/") => {
```

```suggestion
      await page.goto(`http://localhost:3000${base}`);
```
That's because right now other test would fail. If you want to run them locally you can use `npm run test` or `yarn test` or any other package manager.

Reply:
I've made modifications to the code, but on windows, even after attempting to install with both npm and yarn, I consistently encounter the following error when running `npm run test` with or without administrative privileges:

```
TEST  Template "react"
FAIL  Error: EACCES: permission denied, scandir 'D:\vite-express\create-vite-express\templates\react\node_modules\.bin\browserslist'
FAIL  18 failed, 0 passed in 0.047706s
```

Reply:
That's interesting, I'm running linux so that's why I didn't catch it, but thanks for bringing that up. You can push the code so I can take a look and run the CI if everything looks alright.